.github/workflows/eks-tunnel.yaml

```diff
@@ -54,6 +54,11 @@ jobs:
           sudo tar xzvfC eksctl_$(uname -s)_amd64.tar.gz /usr/bin
           rm eksctl_$(uname -s)_amd64.tar.gz
 
+      - name: Install helm
+        uses: azure/setup-helm@18bc76811624f360dbd7f18c2d4ecb32c7b87bab
+        with:
+          version: v3.8.2
```

Comment:
Maybe add a comment here linking to https://github.com/helm/helm/issues/10975 (and optionally a newly created cilium-cli  issue), explaining why we pin to this version specifically. This will hopefully remind us to check again after some time whether the issue is fixed in newer helm versions.